torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -393,6 +448,7 @@ def __init__(
         ignored_params = self._get_ignored_params(self._ignored_modules)
         param_to_unflat_param_names = _get_param_to_unflat_param_names(module)
         self._ignored_param_to_param_name = {}
+        original_module_names = dict(module.named_modules()).keys()
```

Comment:
Where is this used?

Reply:
Was doing some debugging, this isn't used anymore, removing.